demo/index.html

```diff
@@ -100,7 +107,16 @@
                       value: false
                     },
                     selected: String,
-                    steps: Array
+                    steps: Array,
+                    i18n: {
+                      type: Object,
+                      value() {
+                        return {
+                          'OPTIONAL': 'Optional',
+                          'step-3-key': 'Preview'
```

Comment:
This may not be necessary right now. But it looks like this represents i18n key: value mapping. Shouldn't we centralize this into a file that the entire application has access to so that the same keys and values can be reused and translated elsewhere?